src/targetPacks/ILsrc/microsoft.netcore.app/2.1.17/runtime.json

```diff
@@ -0,0 +1,64 @@
+{
+  "runtimes": {
+    "linux-arm": {
+      "Microsoft.NETCore.App": {
+        "runtime.linux-arm.Microsoft.NETCore.App": "2.1.17"
```

Comment:
Hmm. I thought I remembered you mention in standup @dseefeld that we trim the runtime.json at least, but I see the file's still here for the other versions too.

Reply:
Continuing runtime.json inclusion based on precedent seems like a perfectly fine way to continue until we hit a reason otherwise. (Resolving thread.)